README.md

```diff
@@ -1,6 +1,8 @@
 # Rust Space API Implementation
 
-[![Build Status](https://travis-ci.org/coredump-ch/spaceapi-server-rs.svg?branch=master)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
+[![Travis branch](https://img.shields.io/travis/coredump-ch/spaceapi-server-rs/master.svg)](https://travis-ci.org/coredump-ch/spaceapi-server-rs)
```

Comment:
What's the difference between the travis links? And why rename `Build Status` into `Travis branch`